A67-xds-dynamic-parameters.md

```diff
@@ -104,6 +104,9 @@ resources {
 
 *   CNCF xDS proposal
     [TP2: Dynamically Generated Cacheable xDS Resources](https://github.com/cncf/xds/blob/main/proposals/TP2-dynamically-generated-cacheable-xds-resources.md).
+*   gRPC proposal[gRFC A47: xDS Federation](https://github.com/grpc/proposal/blob/master/A47-xds-federation.md).
```

Comment:
I think we can remove the words "gRPC proposal".  Anyone reading this already knows what a gRFC is. :)